src/index.js

```diff
@@ -6,9 +6,8 @@ import App from './App';
 import store from './redux/store';
 
 ReactDOM.render(
-  <React.StrictMode>
```

Comment:
I strongly suggest you keep the `React.StrictMode`, since it protects you from the threats mentioned in this [article](https://www.geeksforgeeks.org/what-is-strictmode-in-react/#:~:text=StrictMode%20is%20a%20React%20Developer,warnings%20for%20its%20child%20components.)